modules/apps/29-fee/ibc_middleware.go

```diff
@@ -85,25 +85,19 @@ func (im IBCMiddleware) OnChanOpenAck(
 	counterpartyChannelID string,
 	counterpartyVersion string,
 ) error {
-	if im.keeper.IsFeeEnabled(ctx, portID, channelID) {
```

Comment:
behaviour recently introduced https://github.com/cosmos/ibc-go/pull/6253/files confused me. If the fee was disabled and became enabled, we wouldn't account for it (it would eventually fail on underlying app). Now I chose to the explicit route